### ISSUE TITLE
Unable to download video data (SSLError: [Errno 1] _ssl.c:1429: error:1408F119:SSL)

### ISSUE THREAD

**Issue by xiaose** [rg3/youtube-dl#7236]
This error does not appear every time. Usually only 1 of 3/4 times and not for all URLs. And this error happend only on Win comp and only in US. If I run it, for example, in Ukraine - no error... may be it country-depends but I'm not sure

$ youtube-dl.exe --verbose https://www.youtube.com/watch?v=KciWtUav65M -f 22
```
[debug] System config: []
[debug] User config: []
[debug] Command-line args: [u'--verbose', u'https://www.youtube.com/watch?v=KciWtUav65M', u'-f', u'22']
[debug] Encodings: locale cp1252, fs mbcs, out cp437, pref cp1252
[debug] youtube-dl version 2015.10.18
[debug] Python version 2.7.8 - Windows-7-6.1.7601-SP1
[debug] exe versions: ffmpeg N-60835-ga3a4d07, ffprobe N-60835-ga3a4d07
[debug] Proxy map: {}
[youtube] KciWtUav65M: Downloading webpage
[youtube] KciWtUav65M: Downloading video info webpage
[youtube] KciWtUav65M: Extracting video information
[youtube] KciWtUav65M: Downloading DASH manifest
[youtube] KciWtUav65M: Downloading DASH manifest
[debug] Invoking downloader on u'https://r7---sn-8xgp1vo-2ia6.googlevideo.com/videoplayback?upn=FzFBW0sT3WY&ipbits=0&pcm2cms=yes&pl=16&source=youtube&signature=7C41B3BA5C113A2894C9567D479BE5DB0681ADA6.647F41BF59BC3D45D3915941E433BC8AC72AD35C&mt=1445274525&mv=m&ms=au&mm=31&mn=sn-8xgp1vo-2ia6&id=o-ALViqRhPxmTohKlEWnlU97DyGfZpq12YxTmr9Gpf1MW5&initcwndbps=2138750&mime=video%2Fmp4&key=yt6&sparams=dur%2Cid%2Cinitcwndbps%2Cip%2Cipbits%2Citag%2Clmt%2Cmime%2Cmm%2Cmn%2Cms%2Cmv%2Cpcm2cms%2Cpl%2Cratebypass%2Crequiressl%2Csource%2Cupn%2Cexpire&ip=71.185.233.154&expire=1445296160&fexp=9405994%2C9408710%2C9408940%2C9413011%2C9413140%2C9414764%2C9416126%2C9417630%2C9417707%2C9418400%2C9418702%2C9419399%2C9422508&lmt=1415825572395523&dur=292.850&itag=22&ratebypass=yes&sver=3&requiressl=yes'
[download] Destination: Alastair James _ Diamonds on the Soles of Her Shoes _ [Paul Simon Cover] - YCG S
essions Demo #3-KciWtUav65M.mp4
[download]  54.2% of 81.98MiB at  3.12MiB/s ETA 00:12 ERROR: unable to download video data: [Errno 1] _s
sl.c:1429: error:1408F119:SSL routines:SSL3_GET_RECORD:decryption failed or bad record mac
Traceback (most recent call last):
  File "youtube_dl\YoutubeDL.pyo", line 1598, in process_info
  File "youtube_dl\YoutubeDL.pyo", line 1540, in dl
  File "youtube_dl\downloader\common.pyo", line 342, in download
  File "youtube_dl\downloader\http.pyo", line 164, in real_download
  File "socket.pyo", line 380, in read
  File "httplib.pyo", line 567, in read
  File "socket.pyo", line 380, in read
  File "ssl.pyo", line 246, in recv
  File "ssl.pyo", line 165, in read
SSLError: [Errno 1] _ssl.c:1429: error:1408F119:SSL routines:SSL3_GET_RECORD:decryption failed or bad record mac
```

**Comment by jaimeMF** [comment]
I doubt this is an error in youtube-dl, it could be your connection or something on YouTube side. If it also fails with DASH formats (`-f bestvideo+bestaudio`), you could try to reproduce it on a browser.

**Comment by dstftw** [comment]
Fixed in 2015.10.24.
